server/src/main/java/org/opensearch/tasks/TaskCancellationService.java

```diff
@@ -82,7 +82,7 @@ private String localNodeId() {
         return transportService.getLocalNode().getId();
     }
 
-    void cancelTaskAndDescendants(CancellableTask task, String reason, boolean waitForCompletion, ActionListener<Void> listener) {
+    public void cancelTaskAndDescendants(CancellableTask task, String reason, boolean waitForCompletion, ActionListener<Void> listener) {
```

Comment:
Why do we need to change the visibility modifier here? We only invoke `TaskManager#cancelTaskAndDescendants` which is already 'public'. Is this something to do with UTs?